pkg/brew/bfs.rb

```diff
@@ -0,0 +1,18 @@
+class Bfs < Formula
+  version '1.0.1'
```

Comment:
Right, I think I get it.  Then yeah I think it makes sense to exclude these files from tarballs in `.gitattributes` so they don't confuse people.